en/02_Developer_Guides/19_GraphQL/03_working_with_generic_types/index.md

```diff
@@ -6,23 +6,21 @@ icon: clipboard
 
 # Working with generic types
 
+> [!NOTE]
+> You are viewing docs for silverstripe/graphql 4.x.
+> If you are using 3.x, documentation can be found
+> [in the GitHub repository](https://github.com/silverstripe/silverstripe-graphql/tree/3)
```

Comment:
Moved this from below to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.